gnosis/cowswap
# Show settings when clicking slippage

# Summary

This PR bring back some functionality that was showing the settings when you click on the slippage.

![image](https://user-images.githubusercontent.com/2352112/127987634-9faa901d-880e-4011-80fa-f11c980310ca.png)



## Context
Before, this functionality was available only if you have the non-auto slippage selected. And there was an issue that was not using the same styles, formatting and tooltip to the customised slippage setting we show. 

#1149 Made the row coherent, but lost this functionality of being able to click on it to open the settings.

This PR brings it back, but also adds the click behaviour in the value of the slippage (not just the label). Also, before we didn't have this behaviour for "Expert mode" where nothing was happening when you clicked in the slippage. This PR also solves that.


# To Test

**Test with auto**
1. Select AUTO slippage and get a price estimation
2. Observe there's no slippage

**Test without auto**
1. Select any slippage and get a price estimation
2. Click in the label and in the value of the slippage
3. Observe the settings is now open

**Test with auto and expert mode**
1. Enable expert, set AUTO-slippage, get a price estimation
2. Click in the label and in the value of the slippage
3. Observe the settings is now open

**Test without auto and expert mode**
1. Enable expert, set any slippage value, get a price estimation
2. Click in the label and in the value of the slippage
3. Observe the settings is now open
